### PR TITLE
Fixed #22289: Dashboard Test workflow

### LINKED ISSUE #22289: Dashboard Test Workflow failing
#### Expected behaviour
The Dashboard Test Workflow should be passing and all tests should successfully pass.

#### Actual behaviour

Workflow fails to execute.

```shell
[13:21:04] Error: Command failed: npx tsc -p test/typescript-dts/dashboards

[13:21:04] Linting failed, make sure you have built the Highcharts declarations first using "npx gulp dist"
[13:21:04] '<anonymous>' errored after 2.14 s
[13:21:04] Error: Command failed: npx tsc -p test/typescript-dts/dashboards
```
**As mentioned, adding an `npx gulp dist` command might fix this issue**

#### Live demo with steps to reproduce
Check [this](https://github.com/highcharts/highcharts/actions/runs/12160554236) workflow run

### PR DESCRIPTION
**Pull request by adityaraute**
Fixes #22289 

Adds a `npx gulp dist` command to the workflow to prevent the Dashboard Test workflow from failing consistently

### CODE REVIEW

File: .github/workflows/dashboards-test.yml [hunk 1]
```diff
@@ -44,9 +44,12 @@ jobs:
 
       - name: Install Dependencies
         run: npm i
-
+        
       - name: Build Highcharts definitions
         run: npx gulp jsdoc-dts
```
Comment (goransle) [inline .github/workflows/dashboards-test.yml:49]: This can likely be removed, as it is included in the main `dist` task

**Review by goransle — COMMENTED** [review]
Thanks! One small improvement:

**Comment by ddragula** [comment]
Hi! Thanks for the contribution. While locally running `npx gulp dashboards/test` requires calling `npx gulp dist` first, on CI it is called before and I don't think there is a need to call it afterwards.

The problem with automation that you sent was probably caused by the lack of copying `d.ts` files in one of the gulptasks, which I fixed here: https://github.com/highcharts/highcharts/pull/22366

It seems to be working properly now.

**Comment by adityaraute** [comment]
Thanks folks for the heads-up. I'll close the PR and the issue, too.

**Pull request closed without merging**